src/app/api/services/dot-router-service.ts

```diff
@@ -70,6 +70,10 @@ export class DotRouterService {
     }
 
     getPortletId(url: string): string {
+        if (url.indexOf('?url=') > 0) {
```

Comment:
When edit page was loading from a redirect in tomcar the url was ```c/site-browser?url=about-us/index```

This method didn't check for ```?url=about-us/index``` so it was returning an invalid porlet id that was making the routing go to first porlet.